fighthealthinsurance/utils.py

```
@@ -49,28 +49,34 @@ def is_convertible_to_int(s):
+        msg = EmailMultiAlternatives(
+            subject,
+            text_content,
+            settings.EMAIL_HOST_USER,
+            [to_email],
+        )
+
+        # Attach the HTML content and send the email
+        msg.attach_alternative(html_content, "text/html")
+        msg.send()
+
+    except Exception as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)